### PR TITLE
Update access-token-passthrough-mode.md

### PR DESCRIPTION
**Pull request by PaulJackson123**
Apply some changes that I discovered in my testing of Snowflake passthrough.

### CODE REVIEW

File: azure/access-token-passthrough-mode.md [hunk 1]
```diff
@@ -71,14 +71,6 @@ For reference, here is a Microsoft [quickstart guide](https://learn.microsoft.co
    > **Note**:
    > If you want to authenticate Launchpad to Azure using a client certificate (instead of a client secret), see [Using a Certificate](./client-certificate-config.md) for details.
 
-3. **Scopes:** Under **Expose an API**, use the default suggested Application ID URI (`api://<client-id>` where `<client-id>` is the **Application (client) ID** noted in Step 1) and add the following scopes. For each scope, set **Admins and users** in the **Who can consent?** option, enter the required display names and descriptions, and then click the **Add scope** button.
```
Comment (paulplace) [inline azure/access-token-passthrough-mode.md:74]: This step is not actually required?

Comment (PaulJackson123) [reply azure/access-token-passthrough-mode.md:74]: Right. I set it up without it (thrice) on my laptop, and can provide a demo.

Comment (paulplace) [reply azure/access-token-passthrough-mode.md:74]: I believe you. And it's nice that this step is not required. I'm just puzzled why we thought it was required.

**Review by paulplace — APPROVED** [review]

File: azure/access-token-passthrough-mode.md (hunk 1 above)
Comment (PaulJackson123) [reply azure/access-token-passthrough-mode.md:74]: I guess we do some extra steps to get it working, especially when it doesn't work at first and we try things until it works. Going back through to eliminate things and test is tedious.

**Pull request merged**